### ISSUE TITLE
Theme issues with culture

### ISSUE THREAD

**Issue by esso23** [skclusive/Skclusive.Material.Component#71]
--theme-typography-h2-font-size, --theme-typography-h4-font-size, etc. has x,yrem (should be x.yrem) on my machine.
The problem with missing InvariantCulture seems to be still present.
I was wondering why the styles are so broken.

**Comment by skclusive** [comment]
thanks for reporting. reproduced the issue and fixed it. planning to release new rc build. i should have named the builds previews.

styles are broken in other locale due to complete rewriting. earlier css values were hardcoded. now font-size and colors are dynamically generated based on base ThemeValue (similar to material-ui implementation)

**Comment by skclusive** [comment]
@esso23 could you please verify in release build (5.0.1)?

**Comment by BrunoZell** [comment]
@skclusive Check out the global invariant switch settable in the project file: https://docs.microsoft.com/en-us/dotnet/core/run-time-config/globalization#examples

```xml
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <InvariantGlobalization>true</InvariantGlobalization>
  </PropertyGroup>

</Project>
```

Maybe this makes it easier to cope with such a behavior in the future. However, I don't know if this switch only applies to the assemblies of the library or to the application as a whole. All documentation seem to indicate that it is global, so it may not be suitable for this library. Further investigation is needed.

Alternatively, setting [`CA1304: Specify CultureInfo`](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1304) as an error could mitigate future issues of this type. This is what I currently do for my server side projects.

**Comment by skclusive** [comment]
@BrunoZell thanks for the references. yes. globally disabling the Invariant might impact app code. the alternate setting looks good. but not sure how to use it. if possible, could you please create PR for the same?

**Comment by skclusive** [comment]
closing the issue and the issue is fixed in 5.0.1 release. please reopen in case reproduced.
